@theme/markdoc/components.tsx

```diff
@@ -212,3 +215,147 @@ export function NotEnabled() {
     <span className="status not_enabled" title={translate("This feature is not currently enabled on the production XRP Ledger.")}><i className="fa fa-flask"></i></span>
   )
 }
+
+// Generate amendments table with live mainnet data
+export function AmendmentsTable() {
```

Comment:
We may want to think about limiting how many amendments we show or provide some pagination in future, as the Amendments table could grow to be quite long. More of a future improvement though.